### DISCUSSION TITLE
👋 Welcome designers! Please introduce yourself

### DISCUSSION THREAD

**Discussion by matthiasdebaat** [home-assistant/frontend#10147] [Other feature requests]
Hello!

It would be great to meet fellow designers and HA enthousiast! Feel free to introduce yourself.
Some questions you may answer to get your thoughts flowing:
- Who are you?
- What is your industry, profession and expertise?
- Where do you like to contribute to Home Assistant?

Welcome!

**Good to know**
We want to make it as easy for designers to contribute as it is for developers. There’s a lot a designer can contribute to. For example making it easier to work with light groups or getting started with automating your house. If you have a design you really think we should look at? Would you like to help build our own Design System? Do you have experience with user testing? Do you have strategic design ideas? Want to organize an online design workshop? Or just want to meet fellow designers?

- You can find the lates UX [discussions](https://github.com/home-assistant/frontend/labels/ux) and [issues](https://github.com/home-assistant/frontend/discussions?discussions_q=label%3Aux) here on GitHub. Everyone can start a new issue or discussion!
- You can also meet us at [devs_ux Discord](https://discord.gg/BPBc8rZ9).
- We're working on design documentation on our [developers website](https://developers.home-assistant.io). In the meantime, you can check out our [Figma DesignKit](https://www.figma.com/community/file/967153512097289521/Home-Assistant-DesignKit).

**Comment by matthiasdebaat** [comment]
Hi! Born in the year of the first The Simpsons episode. I got the chance to turn my hobby into my job to be the first full-time UX designer to step into this beautiful open source project. My goal is to make Home Assistant easier to use and accessible for everyone. By setting up design processes and tooling, doing user research and making designs, I hope to contribute in this way. 

In my browser bookmarks are websites about gadgets, home automation and airsoft. I love the latest technical innovations that make life easier and more fun. And I can regularly be found at various airsoft locations in the Netherlands. Although my camouflage clothes work quite well for not being found.

**Comment by marcelodavanzo** [comment]
Hi Matthias and community 👋 looking forward to helping out with HA in the near future. Here's my intro:

I'm Marcelo, a Product Designer with experience in web and native apps (iOS and Android). Currently work at Trainline (a digital ticket retailer for train and bus tickets) and joining Facebook in December. I have been focusing on Design Systems in the last few years and this is the core of my work right now but also worked on features and enterprise software in the past.

I've been using Home Assistant since Summer 2019 and like many, I now have my house covered in tech and have automations for pretty much anything you can imagine. I've flashed Tasmota, used tuya-convert, created my own devices with ESPHome, write automations in YAML / NodeRed, the works.

**Where do you like to contribute to Home Assistant?**
I'd love to support anything design systems related obviously, but also keen on helping with improving certain areas and flows. I'll be honest and say I'm not quite clear on how contribution from designers will work but I'm more than willing to help figure it out and be a part of it.

**Reply by matthiasdebaat** [reply]
Nice to meet a fellow designer and HA enthousiast! Congrats on your new job! 

Contributing as a designer is also new to us, so we are still learning. One of the things that I think works great is having video calls and discuss certain topics. For example, last Friday we did a call on the Home Assistant menu structure, which gave me input for user research and creating concepts.

If you like we can schedule a call about a Home Assistant design system, what we currently have and I can share some ideas we have. Your experience in design systems could really help us.

**Reply by dshokouhi** [reply]
Awesome to see designers who are familiar with Android! I would also like to say if you need any help with contributing to the Android app we are always open to suggestions and PRs. Myself I am more of a developer and my designs always leave more to be desired but would always be happy to work on updated designs or help others improve upon the existing design :)
